pkg/machine/qemu/options_darwin_arm64.go

```diff
@@ -15,7 +16,7 @@ func (v *MachineVM) addArchOptions() []string {
 		"-accel", "hvf",
 		"-cpu", "cortex-a57",
 		"-M", "virt,highmem=off",
-		"-drive", "file=/usr/local/share/qemu/edk2-aarch64-code.fd,if=pflash,format=raw,readonly=on",
+		"-drive", "file=" + getEdk2CodeFd("edk2-aarch64-code.fd") + ",if=pflash,format=raw,readonly=on",
```

Comment:
Does the Leading " + " matter here?  
Wouldn't this be different then the existing code?

Reply:
Not sure what you mean by "The leading plus".  `getEdk2CodeFd` will return either `/usr/local/share/qemu/edk2-aarch64-code.fd` or `/opt/homebrew/share/qemu/edk2-aarch64-code.fd`.  Those get concatenated in where the full filename used to be.  If it makes things clearer, I can switch to using a formatted string.